src/NIDAQ.jl

```diff
@@ -41,12 +41,12 @@ export Bool32
 
 try
   global ver
-  major = Uint32[0]
-  ccall((:DAQmxGetSysNIDAQMajorVersion,NIDAQmx),Int32,(Ptr{Uint32},),major)
-  minor = Uint32[0]
-  ccall((:DAQmxGetSysNIDAQMinorVersion,NIDAQmx),Int32,(Ptr{Uint32},),minor)
-  update = Uint32[0]
-  ccall((:DAQmxGetSysNIDAQUpdateVersion,NIDAQmx),Int32,(Ptr{Uint32},),update)
+  major = UInt32[0]
```

Comment:
you need to tell julia where the clang install is with
```
ENV["LLVM_CONFIG"]="/full/path/to/your/clang/installation/"
```

not sure where that would be on windows.  halfway down the Clang.jl README are very brief installation instructions.